src/Resources/translations/flashes.pl.yml

```diff
@@ -0,0 +1,5 @@
+bitbag:
+    ui:
+        no_new_shipments_to_export: Brak nowych przesyłek do wyeksportowania.
+        shipment_data_has_been_exported: Dane przesyłce(ach) zostały wyeksportowane.

```

Comment:
Dane przesyłce(ach) -> Dane o przesyłce(ach) ?
or
Dane przesyłce(ach) -> Dane przesyłki(ek) ?

Reply:
Should be `Dane przesyłki(ek)` i will fix it.
In the future we can use ICU message format for pluralization.